GLTFSceneKit/GLTFFunctions.swift

```diff
@@ -213,6 +214,54 @@ func loadImageData(from data: Data) throws -> Image? {
     return Image(data: data)
 }
 
+func saveImageDataToFile(from data: Data, path: URL?, name: String?, mimeType: String?) throws -> String {
+    // Determine file basename
+    /** @see https://www.hackingwithswift.com/example-code/language/how-to-convert-data-to-a-string */
+    let dataStr = String(decoding: data, as: UTF8.self)
+    var basename = SHA256(dataStr).digestString();
+    if let _name = name {
+        basename = "\(_name)_\(basename)"
+    }
+    
+    // Determine file extension
+    var ext: String
+    
+    switch mimeType {
+    case "image/jpeg":
+        ext = ".jpg"
+    case "image/png":
+        ext = ".png"
+    default:
+        ext = ""
+    }
+    
+    let filename = "\(String(describing: basename))\(ext)"
```

Comment:
Why not just this?
```
let filename = "\(basename)\(ext)"
```

Reply:
@gimlids Ah yeah, will do that; `(String(describing:` was left over from debugging.